### PR TITLE
Add Slack link

### PR DESCRIPTION
**Pull request by kampersanda**
This PR adds a Slack link in README. If ok, I'll create the same PR for Vaporetto and Daachorse.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,6 +174,13 @@ EOS
 
 The directory [docs](./docs/) provides descriptions of more advanced usages such as training or benchmarking.
 
+## Slack
+
+We have a Slack workspace for developers and users to ask questions and discuss a variety of topics.
+
+ * https://daac-tools.slack.com/
+ * Please get an invitation from [here](https://t.co/ikroMYhRsm).
```
Comment (vbkaisetsu) [inline README.md:182]: Short links should not be used.
This link depends on the third party service and is no guarantee that it is permanent.

**Review by vbkaisetsu — COMMENTED** [review]
How about adding a badge like this?

[Slack](https://example.com)

**Comment by vbkaisetsu** [comment]
BTW, do we have to renew the invitation link every time it expires?

**Comment by kampersanda** [comment]
@vbkaisetsu We can create an unexpired link as follows:

https://mocotan.hatenablog.com/entry/2020/06/22/113009

I'll create an invitation link in this manner.

However, it seems that 400 members are the maximum for each link ([ref](https://slack.com/intl/ja-jp/help/articles/201330256-%E3%83%AF%E3%83%BC%E3%82%AF%E3%82%B9%E3%83%9A%E3%83%BC%E3%82%B9%E3%81%AB%E6%96%B0%E8%A6%8F%E3%83%A1%E3%83%B3%E3%83%90%E3%83%BC%E3%82%92%E6%8B%9B%E5%BE%85%E3%81%99%E3%82%8B#u25307u24453u12522u12531u12463u12434u20849u26377u12377u12427)).
If reached, the link must be re-created.

File: README.md (hunk 1 above)
Comment (kampersanda) [reply README.md]: fixed, https://github.com/daac-tools/vibrato/pull/131/commits/7b67deebdc43cf79b3b06f1ce97ef39039de8f02

**Review by vbkaisetsu — APPROVED** [review]

**Review by odashi — APPROVED** [review]

**Pull request merged**